packages/lesmis/src/constants/page.js

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
This is obviously pretty basic and wouldn't scale that well if the complexity of the pages were to increase but given the simplicity of the current content pages, I think it works pretty well.